### PR TITLE
update development content

### PR DESCRIPTION
**Pull request by zouyee**
`--kubeconfig`  need to be given

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kube-state-metrics/286)

### CODE REVIEW

**Comment by brancz** [comment]
The `--kubeconfig` flag already exists.

**Comment by andyxning** [comment]
This lgtm.

@brancz WDYT.

File: README.md [hunk 1]
```diff
@@ -166,8 +166,10 @@ metrics right away.
 When developing, test a metric dump against your local Kubernetes cluster by
 running:
 
+> kubeconfig must be set to a valid file(kubeconfig default file name: $HOME/.kube/config)
+
 	go install
-	kube-state-metrics --apiserver=<APISERVER-HERE> --in-cluster=false --port=8080
+	kube-state-metrics --apiserver=<APISERVER-HERE> --in-cluster=false --port=8080 --kube-config=<KUBE-CONIFG-HERE>
```
Comment (brancz) [inline README.md]: The flag is `--kubeconfig` not `--kube-config`

Comment (andyxning) [reply README.md]: Oh. Have not checked that. :)

**Comment by zouyee** [comment]
It has been squashed, I think the description of `kubeconfig` flag may  need to be added though kube-state-metrics running with a default value.

**Comment by brancz** [comment]
IIRC the `--apiserver` flag can be omitted if a kubeconfig is passed.

**Comment by andyxning** [comment]
Yep. If users want to override the apiserver address in kubeconfig, `--apiserver` can  be specified. Others, `--apiserver` can  be omitted. 

@zouyee Could you please add this tips to the doc also. Just under the line:
> kubeconfig must be set to a valid file(kubeconfig default file name: $HOME/.kube/config)

**Comment by andyxning** [comment]
/lgtm 

Will merge when CI is green.

**Pull request merged**